packages/ffe-accordion/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sb1/ffe-accordion",
-  "version": "4.0.12",
+  "version": "4.0.13",
```

Comment:
`lerna` will update this version number automatically, so please don't modify it here